docs/source/user-guide/general/drift_guide.rst

```diff
@@ -0,0 +1,207 @@
+
+Detecting drift is an important warning sign that our model may be not as accurate on the new data compared to the training data, and that it should be
+adjusted or retrained on different data.
+In production environments, detecting drift (and other measured derived from drift, such as model confidence) is often the only way to know that our model performance is deteriorating,
+as in many cases the label is unknown for some time after the prediction is made.
+
+It is important to note that not all changes in data represent drift. For example, periodic changes in data due to daily, weekly or
+seasonal changes are usually not considered drift, as they as they are often present in the training data.
+
+
+Which Types of Drift Are There?
+================================
+
+In machine learning, we usually refer to 3 types of drift:
+
+Concept Drift
+-------------
+Concept drift happens when the underline relation between the data and the label has changed. In layman terms, this means that our
+data has changed in a way that affects our label, meaning a model trained on previous data will have difficulty predicting
+on the new data, as the label can't be predicted in the same way anymore.
+
+Concept drift will almost always require some changes to the model, and will require retraining it on newer data.
+
+Data Drift
+----------
+Data drift is any change in the distribution of the data. It usually refers to changes that do not significantly affect the label.
+However, even when not affecting the model, data drift is important, as it can expose other undetected issues.
+
+Moreover, when labels are not available (as happens in many cases), data drift cannot be discerned from concept drift.
+
+Label Drift
+-----------
+Label drift is the change in the distribution of the label itself. Note that this can also be caused by concept drift,
+but here we discuss a change that means that the label can still be accurately predicted from the data, but its
+distribution has changed.
+
+In many cases, label drift alone might not be of interest. However, it may affect your model's accuracy, as less-accurate
+classes may be more prevalent now that they were in your train dataset.
+
+For more on the different types of drift, `see here <https://deepchecks.com/data-drift-vs-concept-drift-what-are-the-main-differences/>`_
+
+What Can You Do in Case of Drift?
+---------------------------------
+
+There are several solutions to distribution drift. They are also dependent on the information you have.
+
+* If your drifted data has labels, you can easily know which type of drift you are facing.
+* However, the more common case is when you don't have your data labels yet. In this case, we can only try and measure
+  drift on the data (features) or the model's output.
+  If we detect drift, we have no way to know whether this drift is a simple data drift, label drift or concept drift, as
+  we don't know the true labels.
+
+(Feels disconnected to the rest here)
```

Comment:
Maybe drop a note about how looking at the drift of the highest FI features may help understand if we should worry or not?